setup.cfg

```diff
@@ -41,7 +41,6 @@ tests_require =
 
 [options.extras_require]
 slugify =
```

Comment:
sorry for asking, but is there a reason to keep the empty extra requirement?

Reply:
The intention was to not break the installation right away in case someone was using `slugify` as extra dependency. But maybe it is actually better to get rid of it completely.

@nschlemm is this causing you issues, or it is just your observation?